src/conf/local-file.yaml

```diff
@@ -44,6 +44,14 @@ publisher:
   enable: true
   cron-job-time-interval: "0h1m00s"         # format: XhYmZs
 
+purge-old-db-entries:
+  enable: true 
+  cron-job-time-interval: "10d0h0m00s"      # format: WdXhYmZs
+  dbname: ./accuknox-obs.db
```

Comment:
@yasin-cs-ko-ak 
Please make this configuration to take multiple db names. String array. 
You can take reference from nsFilter and nsNotFilter.

Reply:
Changed to string array.